projects/approved-premises-and-delius/src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/MessagingIntegrationInactiveTest.kt

```diff
@@ -150,7 +150,13 @@ internal class MessagingIntegrationInactiveTest {
         assertThat(contact.description, equalTo("Approved Premises Booking for Hope House"))
         assertThat(
             contact.notes,
-            equalTo("To view details of the Approved Premises booking, click here: https://approved-premises-dev.hmpps.service.justice.gov.uk/applications/484b8b5e-6c3b-4400-b200-425bbe410713")
+            equalTo(
+                """
+                Current: 30/01/2023 to 30/04/2023
```

Comment:
Has this format come from somewhere, or is it a best guess?

Just wondering if we should make it clearer/more consistent with other generated notes fields e.g. something like:
```suggestion
                Expected arrival: 30/01/2023
                Expected departure: 30/04/2023
```

Reply:
I took it from Booking changed note format.

Reply:
I will make the change to separate the fields.